config/sample-frontend-config.yaml

```diff
@@ -22,9 +22,13 @@ quickFilters:
   - name: Services network
     filter:
       dst_kind: 'Service'
+  - name: DNS
```

Comment:
hmm how does that work if the feature is off? Is it ignored?

Reply:
That's just the sample. The real config comes from operator
https://github.com/netobserv/network-observability-operator/blob/7e7a9f88f1612354d3fa98cbb4d558f2bddc3b74/controllers/consoleplugin/consoleplugin_objects.go#L313-L318